libs/core/src/lib/command/index.ts

```diff
@@ -208,14 +205,11 @@ export class Command {
   configureProperties() {
     const { concurrency, sort, maxBuffer } = this.options;
 
-    this.concurrency = Math.max(1, +concurrency || DEFAULT_CONCURRENCY);
+    this.concurrency = Math.max(1, +(concurrency ?? 0) || DEFAULT_CONCURRENCY);
```

Comment:
What additional case are you attempting to protect against here?

I believe it would already always do the right thing (given `NaN` and `0` are falsely values, and the result of coercion for values such as `undefined` and `null`), so I think it would be simpler to revert this unless there is something I am missing

Reply:
Typechecker is reporting an error because it is possibly undefined from the defined options. I gave it a default value during assignment

Reply:
Typechecker is reporting an error because concurrency can possibly be null from options. So I assigned a default value to concurrency so it is always set even if undefined from options.